services/keybase/keybase-profile.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+
+module.exports = class KeybaseProfile extends BaseJsonService {
+  static get apiVersion() {
+    throw new Error(`apiVersion() is not implemented for ${this.name}`)
+  }
+
+  async fetch({ schema, options }) {
+    const apiVersion = this.constructor.apiVersion
+    // See https://keybase.io/docs/api/1.0/call/user/lookup.
+    const url = `https://keybase.io/_/api/${apiVersion}/user/lookup.json`
+
+    return this._requestJson({
+      url,
+      schema,
+      options,
+    })
+  }
+}
```

Comment:
Add a `transform` function to the base class to encapsulate the invalid/unknown username checks:
```js
  transform({ data }) {
    if (data.status.code !== 0) {
      throw new NotFound({ prettyMessage: 'invalid username' })
    }

    if (data.them.length === 0 || !data.them[0]) {
      throw new NotFound({ prettyMessage: 'profile not found' })
    }

    return { user: data.them[0] }
  }
```